app/presenters/two_factor_options_presenter.rb

```diff
@@ -68,6 +68,30 @@ def phone_options
     [TwoFactorAuthentication::PhoneSelectionPresenter.new]
   end
 
+  def sms_option
+    if piv_cac_required? ||
+       aal3_only? ||
+       IdentityConfig.store.hide_phone_mfa_signup ||
```

Comment:
Should we extract some of these repeated conditions (as well as in `phone_options`) to a helper method? e.g. `def phone_allowed?`